src/proof_system/widget/mod.rs

```diff
@@ -29,20 +29,23 @@ pub struct ProverKey {
     pub logic: logic::ProverKey,
     /// ProverKey for range gate
     pub range: range::ProverKey,
-    /// ProverKey for fixed base curve addition gates
+    /// ProverKey for lookup gates
```

Comment:
This description change isn't correct. This `proverKey` still refers to the curve ops.